content/docs/latest/setup/customization/configuring-dns.md

```diff
@@ -60,6 +60,29 @@ storage:
 ```
 This is known to interfere with [Kubernetes][kubernetes] in certain situations.
 
+## Special considerations regarding Multicast DNS (mDNS)
+
+In some network environments, the reserved top level domain `.local` is used to identify internal devices.
+
+This is in contradiction with [RFC 6762](https://datatracker.ietf.org/doc/html/rfc6762) but could be difficult to change across an entire environment.
+
+In effect, you will get `SERVFAIL` response from `resolved` when trying to resolve names ending with `.local`.
+
+A simple workaround is to add `local` or `yourcompany.local` as a search domain in your network configuration. Here is an example in a `systemd-networkd` configuration file :
+
+```
```

Comment:
Looks good. Maybe for consistency with the above documentation, can you please add this change as a butane configuration?

Reply:
```yaml
variant: flatcar
version: 1.0.0
storage:
  files:
    - path: /etc/systemd/network/10-nabox.network
      overwrite: true
      mode: 0644
      contents:
        inline: |
          # /etc/systemd/network/10-nabox.network:
          [Match]
          Name=ens192

          [Network]
          LinkLocalAddressing=no
          IPv6AcceptRA=no
          Address=10.1.1.60/16
          Gateway=10.1.0.1
```